core/src/test/java/de/jplag/RootFolderTest.java

```diff
@@ -0,0 +1,73 @@
+package de.jplag;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.File;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import de.jplag.exceptions.ExitException;
+
+/**
+ * Test class for the multi-root feature and the old-new feature.
+ * @author Timur Saglam
+ */
+class RootFolderTest extends TestBase {
+
+    private static final String BASECODE_SUBMISSION = "base";
+    private static final String GLOBAL_BASECODE = "basecode-base";
+    private static final String BASE_SAMPLE = "basecode";
+    private static final String SIMPLE_SAMPLE = "SimpleDuplicate";
+
+    @Test
+    void testMultiRootDirNoBasecode() throws ExitException {
+        List<String> paths = List.of(getBasePath(BASE_SAMPLE), getBasePath(SIMPLE_SAMPLE)); // 3 + 2 submissions.
```

Comment:
Instead of the comments, why not use an assert /precondition to communicate the number of expected files? This would also help identify incorrect preconditions

Reply:
These tests were copied from another test class, as tests were split up. I think these comments explain the input scenario (folder/file structure of the sample), but I am not sure.

Reply:
Would understand them like. While thinking of it, maybe a low overhead refactoring while improving readability a lot would be to extract those numbers into constants (like `BASE_SAMPLE_COUNT`, `SIMPLE_SAMPLE_COUNT`, etc) and computing the expected numbers based on that?